Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -24,6 +24,8 @@ export interface Spec extends TurboModule {
       +tintColor?: ?number,
       +userInterfaceStyle?: ?string,
       +disabledButtonIndices?: Array<number>,
+      +icons?: ?Array<Object>, // TODO: This should be a ResolvedAssetSource
```

Comment:
do you plan to land this code with this comment? it's not totally clear here

Reply:
I do, because I wasn't sure how to correctly type this and there is no documentation for Turbo Modules that's publicly available yet. If there is a better way to do this, then feel free to either let me know or add a commit to this branch.

Reply:
Hey @matt-oakes!

In the TurboModule codegen, you can use any Flow types that you declare in the Spec. So, if you know the shape of ResolvedAssetSource, you can do this:

```
type ResolvedAssetSource = {
  foo: string,
};

icons: ?Array<ResolvedAssetSource>
```

Reply:
@RSNara Can you point me to a place in the code where the shape of this object is defined, or another place where it's passed through JSI?

At the time I wrote this, the other places that were passing asset references were doing what I did, however, that might have changed now.